podcasts/PCTableViewController.swift

```diff
@@ -0,0 +1,121 @@
+import UIKit
+
+/// Allows for dynamic creation of reusable table cells.
+/// By default the `reuseIdentifier` is equal to the name of the cell's class
+/// and the nib is loaded from a file with the same name
+///
+protocol ReusableTableCell {
```

Comment:
[nit] I think in the end this is a `ReusableNib`. Anyway, for the context of this task I think it's ok to leave it as it is.